### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by nartes**
1. Rubocop gem was installed via bundler. Also auto configuration were performed. And after some options were altered.

2. All issues detected by Rubocop was fixed in the next commit.

UPD: #1

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 51
+# Configuration parameters: AllowHeredoc, AllowURI, URISchemes.
+# URISchemes: http, https
+Metrics/LineLength:
+  Max: 130
```
Comment (kislak) [inline .rubocop_todo.yml]: It is hard to read long string - good practice is to use less than 80 characters in  a string

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    - 'app/helpers/application_helper.rb'
+    - 'app/helpers/manufacturers_helper.rb'
+    - 'app/helpers/products_helper.rb'
+    - 'app/helpers/transports_helper.rb'
+    - 'app/models/manufacturer.rb'
+    - 'app/models/product.rb'
+    - 'config/application.rb'
+    - 'db/migrate/20160401080634_create_transports.rb'
+    - 'db/migrate/20160401080746_create_manufacturers.rb'
+    - 'db/migrate/20160401081904_create_products.rb'
```
Comment (kislak) [inline .rubocop_todo.yml]: I would rather disable the feature (use '*/*') - so many entries pollute the code.

**Comment by kislak** [comment]
You should not close requests unless there is a good reason for it. (like feature is not required  any more)
it is better to fix origin request - so that the context of negotiation is not lost.

**Comment by nartes** [comment]
@kislak, please, checkout updates!

File: .rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  Exclude:
+    - db/schema.rb
+    - bin/*
+Metrics/MethodLength:
+  Max: 30
```
Comment (kislak) [inline .rubocop.yml]: We use 5 but I suppose 7 is ok here

**Comment by kislak** [comment]
Looks good.

**Comment by nartes** [comment]
Should i try to achieve 7 lines per method? Because i have no idea how to split that code
http://pastebin.com/ULWpEsA8.

**Comment by kislak** [comment]
controller may look like this:

MyController
  respond_to :html, :json

  def create
   @product = Product.create(product_params)
     respond_with @product
  end
end

the rest should be in view

2016-04-06 18:58 GMT+03:00 Sergei Sinyak <notifications@github.com>:

> Should i try to achieve 7 lines per method? Because i have no idea how to
> split that code
> http://pastebin.com/ULWpEsA8.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nartes/distributer-shop/pull/3#issuecomment-206440996>
>



-- 
Sincerely Yours,
Serge Kislak

mobile: +375257404426
skype: kislak7

**Comment by nartes** [comment]
I will create separate issue - #7 "7 lines per method refactoring". And this request will be merged.

**Pull request merged**